### ISSUE TITLE
OAuth logins trigger email verification

### ISSUE THREAD

**Issue by michael-robbins** [vabene1111/recipes#695]
### Version
Please provide your current version (can be found on the system page since v0.8.4)
Version: 0.16.2

### Bug description
When a user logs in through the oauth flow (I'm using Keycloak for example). The email verification is triggered anyway, even through my IDP has done this for me.

Is there a way to assume emails coming from oauth logins are verified automatically?

**Comment by vabene1111** [comment]
you can simply not configure SMTP credentials and no emails will be send :) 

But i will look into this behavior

**Comment by michael-robbins** [comment]
Yeah mad, so I haven't configured it, users who authenticate through oauth still get a 'you have been sent a verification email' popup too, so that might be confusing as well for them

**Comment by vabene1111** [comment]
thats interesting but makes sense .. sadly django-allauth (which is what the auth system builds upon) has very limited options for configuration in regards to email sending ...

i will look into this and try to find a solution
